src/components/Input/Input.js

```diff
@@ -1,64 +1,35 @@
 import React, {Component, PropTypes} from 'react';
-import Field from '../Field/Field';
-import $ from 'jquery';
 import classNames from 'classnames';
 
-class Input extends Component {
+export default class Input extends Component {
   static propTypes = {
+    className: PropTypes.string,
     dataContent: PropTypes.string,
     defaultValue: PropTypes.string,
     iconClass: PropTypes.string,
-    label: PropTypes.string,
-    name: PropTypes.string,
-    onChange: PropTypes.func,
     ref: PropTypes.string,
-    small: PropTypes.string,
-    transparent: PropTypes.bool,
-    type: PropTypes.string,
-    value: PropTypes.string,
   };
 
   static defaultProps = {
-    ref: 'popper',
     type: 'text',
   };
 
-  componentWillMount() {
-    this.elm = $(React.findDOMNode(this.refs[this.props.ref]));
-    if (this.props.dataContent) {
-      this.elm.popup({
-        on: 'focus',
-      });
-    }
-  }
-
-  handleChange = (e) => {
-    this.props.onChange(e);
-  };
-
   render() {
     let icon = <i className={this.props.iconClass} />;
     let classes = classNames(
       'sd-input',
       'ui',
-      {small: this.props.small},
-      {transparent: this.props.transparent},
+      this.props.className,
       {icon: !!this.props.iconClass},
       'input'
     );
+    let inputProps = _.clone(this.props);
+    delete inputProps.className;
```

Comment:
We only want to spread non class props on the input, the classes are spread on the containing div.